src/main/java/com/puppycrawl/tools/checkstyle/utils/ModuleReflectionUtil.java

```diff
@@ -46,21 +46,21 @@ private ModuleReflectionUtil() {
     }
 
     /**
-     * Gets checkstyle's modules (directly, not recursively) in the given packages.
+     * Gets valid checkstyle's classes (directly, not recursively) in the given packages.
      *
      * @param packages the collection of package names to use
      * @param loader the class loader used to load Checkstyle package names
-     * @return the set of checkstyle's module classes
+     * @return the set of checkstyle's classes
      * @throws IOException if the attempt to read class path resources failed
      * @see #isCheckstyleModule(Class)
      */
-    public static Set<Class<?>> getCheckstyleModules(
+    public static Set<Class<?>> getValidCheckstyleClasses(
```

Comment:
I would keep a name getCheckstyleModules and isCheckstyleModule.

getValidCheckstyleClasses and isValidCheckstyleClass are a bit weird names.

Reply:
What do you want `isValidCheckstyleClass` named? I still think we should keep `isCheckstyleModule`.

Reply:
https://github.com/checkstyle/checkstyle/pull/11644#discussion_r1008018002

Reply:
Done.